App.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect } from "react";
+import { BrowserRouter as Router, Routes, Route, Navigate } from "react-router-dom";
+import LogIn from "./pages/LogIn";
+import Sidebar from "./components/sideBar/Sidebar";
+import SignUp from "./pages/SignUp";
+import Home from "./pages/Home";
+import Bikes from "./pages/Bikes";
+import Details from "./pages/Details";
+import Reservations from "./pages/Resevation";
+import "./index.css";
+import AddBike from "./pages/AddBike";
+import DeleteBike from "./pages/DeleteBike";
+import MyReservations from "./pages/MyReservations";
+import sessionStorage from "./helpers/sessions";
+import Error from "./pages/404Page";
+import { useGetBikesQuery } from "./components/api/apiSlice";
+import Spiner from "./reusables/spiner/Spinner";
+
+function App() {
```

Comment:
- [ ] The `components` folder seems to be duplicated in the project:

![image](https://github.com/soulemanou-software/car-rental-frontend/assets/57421823/a798a954-4334-41f9-93e0-351c15daa93e)

A "components" folder exists with the same files both on the root, and on "src" folders.

Kindly remove the one that you are not using, if not using it 👍